packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -569,6 +570,40 @@ export async function deleteNamespace(
   });
 }
 
+export async function getInviteInfo(
+  req: AuthRequest<unknown, { key: string }>,
+  res: Response
```

Comment:
Do you think it would be a good idea for us to type the response, e.g.:

```ts
  res: Response<
    | { status: 200; organization: string; role: MemberRole }
    | { status: number; message: string }
  >
```